content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -167,13 +194,29 @@ and installing the sidecar injector webhook configuration on the remote cluster
         pilot:
           configMap: true
         istiodRemote:
-          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/:ENV:cluster=${REMOTE_CLUSTER_NAME}:ENV:net=network1
+          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/cluster/${REMOTE_CLUSTER_NAME}/net/network1
         base:
           validationURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/validate
     EOF
     {{< /text >}}
 
-    Then, install the configuration on the remote cluster:
+1.  If you are using an IP address for the `EXTERNAL_ISTIOD_ADDR`, instead of a proper DNS hostname,
+    modify the configuration to specify the discovery address and paths, instead of URLs:
+
+    {{< warning >}}
+    This is not recommended in a production environment.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ sed  -i'.bk' \
+      -e "s|injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017|injectionPath: |" \
```

Comment:
why we need to do this?

Reply:
the diffrence of two files:
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  namespace: external-istiod
spec:
  profile: external
  values:
    global:
      istioNamespace: external-istiod
      remotePilotAddress: 1.1.1.1
      configCluster: true
    pilot:
      configMap: true
    istiodRemote:
      injectionPath: /inject/cluster/aaaaaa/net/network1
```


```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  namespace: external-istiod
spec:
  profile: external
  values:
    global:
      istioNamespace: external-istiod
      configCluster: true
    pilot:
      configMap: true
    istiodRemote:
      injectionURL: https://1.1.1.1:15017/inject/cluster/aaaaaa/net/network1
    base:
      validationURL: https://1.1.1.1:15017/validate
```

Reply:
Since we don't have a proper secure DNS hostname, we are changing the config to generate a "proxy" istiod kubernetes service that will be used in the webhooks. The `remotePilotAddress` is used to configure the endpoint for the proxy service (if you run `istioctl manifest generate`, you can see the generated service and endpoint in the output yaml).

Reply:
We have to do this step? It seems that no error will be reported if this step is not performed.

Reply:
Without this step, the certs that istiod patches will not work.

I believe we can make it work without this step if we instead set `values.pilot.env.ISTIOD_CUSTOM_HOST="${EXTERNAL_ISTIOD_ADDR}` when we install istiod on the external cluster,  but I don't know whether or not that is a better approach.

Reply:
cc @stevenctl